docs/narr/i18n.rst

```diff
@@ -270,12 +270,12 @@ system run this command:
    $ sudo apt-get install gettext
 
 Installing Lingua is done with the Python packaging tools. If the
-:term:`virtualenv` into which you've installed your :app:`Pyramid` application
-lives in ``/my/virtualenv``, you can install Lingua like so:
+:term:`virtual environment` into which you've installed your :app:`Pyramid`
+application lives in ``/my/venv``, you can install Lingua like so:
 
 .. code-block:: text
 
-   $ cd /my/virtualenv
+   $ cd /my/venv
    $ $VENV/bin/easy_install lingua
```

Comment:
This looks to be the one place where easy_install is still mentioned. Was that on purpose (such as lingua having some odd incompatibility with pip), or just an omission?